packages/scalecube-discovery/src/helpers/utils.ts

```diff
@@ -0,0 +1,3 @@
+export const isNodeEnv = () => typeof window === 'undefined';
+
+export const getGlobal = () => isNodeEnv() ? global : window;
```

Comment:
see my earlier comment 
I would implement it: window ==> global ==> WorkerGlobalScope ==> {}